proposed/2022/DotnetListPackageMachineReadableJsonOutput.md

```diff
@@ -772,6 +772,172 @@ Outputs json for format version 1, if it's not specified then latest version'll
 }
 ```
 
+### Error handling
+
+In case of an error it would be written into `problems` section and return non-zero error exit code to indicate there had been some error to help scripting use cases.
+For current implementation we'll returns general error code `1` from application, more specific error code for each error scenario is not considered in this spec scope.
```

Comment:
A way to word this in a more generic fashion would be to say, `non-0 exit code` instead.

Reply:
Sure, resolved.